frontend/public/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "livepowerhour.com",
+  "name": "songbook generator",
```

Comment:
Doesn't matter for now, but before we suggest people install the app we should review the [icon guidelines](https://web.dev/learn/pwa/web-app-manifest/) and probably title case the name

Reply:
Noted!